docs/src/main/asciidoc/redis-reference.adoc

```diff
@@ -136,7 +136,7 @@ public class RedisExample {
 
 TIP: When using `@RedisClientName`, you can omit the `@Inject` annotation.
```

Comment:
```suggestion
TIP: You can omit the `@Inject` annotation when using `@RedisClientName`.
```

I would change the phrasing here to be more direct. I think the rule otherwise will be loosened, as your word choice here is appropriate.